SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -42,11 +45,24 @@ FORCEINLINE UClass* ResolveClass(FString& ClassPath)
 
 void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 {
-	checkf(IsSupportedClass(Class), TEXT("Could not find class in schema database: %s"), *Class->GetPathName());
 
 	TSharedRef<FClassInfo> Info = ClassInfoMap.Add(Class, MakeShared<FClassInfo>());
 	Info->Class = Class;
 
+	if (!IsSupportedClass(Class))
```

Comment:
This should be moved to the top of the function.

Reply:
In the method that calls this one we rely on the `Class` being present in the `ClassInfoMap`  (see [here](https://github.com/spatialos/UnrealGDK/blob/474c150fc70f76c4043a5c7dfc4b668f31ecf861/SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp#L186)), therefore we need to execute those two lines.

Reply:
Oh are you saying if it's not added to the ClassInfoMap we crash?

Reply:
Yes, exactly.

Reply:
Will add a comment.